### ISSUE TITLE
Question about State of JS 2024

### ISSUE THREAD

**Issue by Justineo** [Devographics/surveys#267]
Hello, I have some questions about the data in the charts from the State of JS 2024 report.

Taking Vue.js as an example, the line chart shows that the Positivity Value has been 65%, 63%, 63%, and 70% from 2021 onwards, respectively:

<img width="756" alt="image" src="https://github.com/user-attachments/assets/7a5a8647-4db6-46be-8b33-8cb48b1354e4" />

However, in the 2D trend chart, it appears that the score for Positive opinions in 2024 is even lower than in 2021:

<img width="687" alt="image" src="https://github.com/user-attachments/assets/83af9a0c-c3aa-4f3b-9e3c-a391ba6fe97a" />

Could someone clarify whether these two types of charts use the same data?

**Comment by SachaG** [comment]
Yes, they use slightly different calculations. The bottom chart is due for a rewrite to harmonize it with the rest of the survey.

**Comment by Justineo** [comment]
The difference is quite confusing, especially when in some cases the trend in one chart is improving while in another chart it is worsening.
